catalog-info.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: go-logger
   annotations:
     github.com/project-slug: coopnorge/go-logger
+    backstage.io/techdocs-ref: dir:.
```

Comment:
Is this notation correct?

Reply:
yes, see https://github.com/coopnorge/go-datadog-lib/blob/39e468e400b74e4db062664bc719113e56292c24/catalog-info.yaml#L11 for a different example.